filters/general/filter_34_HageziMultiNormal/configuration.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "HaGeZi Personal Black & White",
-  "description": "An extension for existing blocklists, my personal black and whitelist. Unblocks false positives and referral domains.",
+  "name": "HaGeZi Multi NORMAL",
```

Comment:
In the filter's meta - `HaGeZi's Normal DNS Blocklist`

Reply:
@Alex-302 
I take it that the author of the filter wants such a title:
https://github.com/AdguardTeam/HostlistsRegistry/pull/272

Reply:
![multi_normal](https://github.com/AdguardTeam/HostlistsRegistry/assets/18536328/e612cb17-f160-4848-9f62-391212209e88)